model/observed/peeragent.go

```diff
@@ -0,0 +1,55 @@
+package observed
+
+import (
+	"context"
+	"time"
+
+	"go.opencensus.io/tag"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/label"
+
+	"github.com/filecoin-project/sentinel-visor/metrics"
+	"github.com/filecoin-project/sentinel-visor/model"
+)
+
+type PeerAgent struct {
+	//lint:ignore U1000 tableName is a convention used by go-pg
+	tableName struct{} `pg:"observed_peer_agents"`
+
+	// ObservedAt is the time the observation was made
+	ObservedAt time.Time `pg:",notnull"`
+
+	// RawAgent is the raw peer agent string
+	RawAgent string `pg:",notnull"`
+
+	// NormalizedAgent is a parsed version of peer agent string, stripping out patch versions
+	NormalizedAgent string `pg:",notnull"`
+
+	// Count is the number of peers with the associated agent
+	Count int64 `pg:",use_zero,notnull"`
```

Comment:
I wonder if there is a way to make these code comments table comments on the model via go-pg :thinking:

Reply:
I don't think so but we could/should be generating our schema from these structs and we could do some magic with turning Go comments into postgres comments